client/me/security-2fa-enable/index.jsx

```diff
@@ -178,22 +178,54 @@ module.exports = React.createClass( {
 		}
 	},
 
+	getToggleLink: function() {
+		return (
+			<a
+				className="security-2fa-enable__toggle"
+				onClick={ function( event ) {
+					analytics.ga.recordEvent( 'Me', 'Clicked On Barcode Toggle Link', 'current-method', this.state.method );
+					this.toggleMethod( event );
+				}.bind( this ) }
+			/>
+		);
+	},
+
 	renderQRCode: function() {
 		return (
-			<div>
-				<p>
-					{ this.translate( 'Scan this QR code with your mobile app:' ) }
+			<div className="security-2fa-enable__qr-code-block">
+				<p className="security-2fa-enable__qr-instruction">
+					{
+						this.translate(
+							"Scan this QR code with your mobile app. {{toggleMethodLink}}Can't scan the barcode?{{/toggleMethodLink}}", {
+								components: {
+									toggleMethodLink: this.getToggleLink()
+								}
+							}
+						)
+					}
 				</p>
-				{ this.state.otpAuthUri ? <QRCode value={ this.state.otpAuthUri } size={ 150 } /> : null }
+				<div className="security-2fa-enable__qr-code">
```

Comment:
Good question. I made the decision to show the div that way there wasn't a reflow after the QR code came back from the API. So, in short, the div acts as a placeholder of sorts.

Perhaps, we should show a placeholder animation there? cc @rickybanister